features/admin.logs.v1/components/remote-logging-config-form.tsx

```diff
@@ -0,0 +1,461 @@
+                                        className="text-field-container mt-3"
+                                        ariaLabel="username"
+                                        required={ false }
+                                        data-componentid={ `${componentId}-username` }
+                                        name="username"
+                                        type="text"
+                                        label={ t("console:manage.features.serverConfigs." +
+                                            "remoteLogPublishing.fields.advanced.basicAuthConfig." +
+                                            "serverUsername.label") }
+                                        placeholder={ t("console:manage.features.serverConfigs." +
+                                            "remoteLogPublishing.fields.advanced.basicAuthConfig." +
+                                            "serverUsername.placeholder") }
+                                        component={ TextFieldAdapter }
+                                        maxLength={ 100 }
+                                        minLength={ 0 }
+                                    />
+                                    <FinalFormField
+                                        key="password"
+                                        FormControlProps={ {
+                                            margin: "dense"
+                                        } }
+                                        width={ 16 }
+                                        className="text-field-container mt-3"
+                                        ariaLabel="password"
+                                        required={ false }
+                                        data-componentid={ `${componentId}-password` }
+                                        name="password"
+                                        type="password"
+                                        label={ t("console:manage.features.serverConfigs." +
+                                            "remoteLogPublishing.fields.advanced.basicAuthConfig." +
+                                            "serverPassword.label") }
+                                        placeholder={ t("console:manage.features.serverConfigs." +
+                                            "remoteLogPublishing.fields.advanced.basicAuthConfig." +
+                                            "serverPassword.placeholder") }
+                                        component={ TextFieldAdapter }
+                                        maxLength={ 100 }
+                                        minLength={ 0 }
+                                    />
+                                    <Button
+                                        size="medium"
+                                        variant="contained"
+                                        onClick={ handleSubmit }
+                                        className={ "button-container mt-3" }
```

Comment:
It is not recommended to use utility classes anymore. Let's move this to the related scss file. 

Check other places as well.